fixed accordion expand/collapse functionality to work as expected. Al…

prd409-utsa opened the pull request:
…so added whit underline underneath the collapsed accordions to show separation.

lalov1 commented on components/03-sections/01-accordion/accordion.hbs:
```diff
@@ -1,15 +1,22 @@
+                                {{text}}
+                            </p>
                             <ul class="ul-list">
                                 <li class="list-item">Use the HTML <ol> element to define an ordered list</li>
```
Please add backticks (``) around `<ol>` element, so it doesn't cause a code break.

prd409-utsa commented:
[Issue #281](https://github.com/utsa-asc/college-dls/issues/281) - add bottom border between collapsed accordions - is also fixed in this PR

[Issue #301](https://github.com/utsa-asc/college-dls/issues/301) - remove forced p and li font-size - is also fixed in this PR

lalov1 commented on components/03-sections/01-accordion/accordion.hbs:
```diff
@@ -22,15 +29,58 @@
+                                <li class="list-item">Use the HTML <ol> element to define an ordered list</li>
+                                <li class="list-item">Use the HTML type attribute to define the numbering type</li>
+                                <li class="list-item">Use the HTML
+                                <li> element to define a list item</li>
```
Please add backticks (``) around `<li>` element, so it doesn't cause a code break.

lalov1 commented on components/03-sections/01-accordion/accordion.hbs:
```diff
@@ -1,15 +1,22 @@
+                </h5>
+                <div id="collapse-01" class="accordion-collapse collapse" data-bs-parent="#accordion">
                     <div class="card-body">
                         <div class="accordion-body-content">
```
The content inside the body looks a little tight and might need a little padding. Maybe add a `p-3` class or consult with @Uri-E or @garza-utsa on their thoughts.

prd409-utsa replied:
I added these styles from orginal scss for **.accordion-body-content** in last commit.  It was originally named **.accordion-body-cntnt** and I missed it when I change cntnt to content for all other accordion selectors. So it wasn't picking it up first time around.  

Long story short.

this was fixed in the last commit I made.

lalov1 commented on components/03-sections/01-accordion/accordion.scss:
```diff
@@ -1,95 +1,149 @@
 }
-.accordion-body-content p {
+
+#accordion .accordion-body-content p {
```
Should this be removed according to #301 ?

prd409-utsa replied:
No. otherwise it doesn't pick up the other styles for the accordion paragraph of 
.accordion-body-content p {
    color: #0c2340;
    margin-bottom: 30px;
    margin-top: 30px;
}

lalov1 reviewed (changes requested):
I added a couple of minor edits. Would you be able to do those before merging the code?

Uri-E reviewed (commented):
Could we achieve the same separation with a margin-bottom:3px? It might be better just in case the accordions are ever used on a non-white background. I do like the idea tho, it looks better to not have them bunched up.

prd409-utsa commented:
Nice suggestion @Uri-E!  It works beautifully. I think 2px width is better though. 3px was too much space...in my opinion.

<img width="1392" alt="Screen Shot 2022-06-21 at 9 50 53 AM" src="https://user-images.githubusercontent.com/92815346/174830518-d75c88c7-b434-4078-8d1a-8ae7b0447cfd.png">

prd409-utsa commented:
Okay, @garza @lalov1 @Uri-E  I just made a commit to make the changes from your code reviews.  Let me know if I missed anything.

@Uri-E here is screenshot showing the background color behind accordion showing through between them
<img width="1319" alt="Screen Shot 2022-06-21 at 10 35 06 AM" src="https://user-images.githubusercontent.com/92815346/174843072-db2783c5-c48f-4672-8e5a-8486e2bd9216.png">

lalov1 commented on components/03-sections/01-accordion/accordion.scss:
```diff
@@ -1,95 +1,143 @@
 }
-.accordion-body-content ul li {
+
+.accordion .accordion-body-content ul li {
```
I know it was mentioned in #301 that the font-size for the li is larger than the p tag. Maybe we should remove the font-size for both elements and use the default size?

lalov1 reviewed (commented):
It looks like the `<li>` and `<p>` have different sizes in the `.accordion .accordion-body-content`, which was mentioned in #301. Should we have them the same size or remove the `font-size` so they could use the default size?
